nav2_behaviors/include/nav2_behaviors/timed_behavior.hpp

```diff
@@ -224,10 +225,10 @@ class TimedBehavior : public nav2_core::Behavior
 
     ResultStatus on_run_result = onRun(action_server_->get_current_goal());
     if (on_run_result.status != Status::SUCCEEDED) {
-      RCLCPP_INFO(
-        logger_,
-        "Initial checks failed for %s", behavior_name_.c_str());
       result->error_code = on_run_result.error_code;
+      result->error_msg = "Initial checks failed for " + behavior_name_ + " - " +
+        on_run_result.error_msg;
+      RCLCPP_INFO(logger_, result->error_msg.c_str());
```

Comment:
I understand this for a log, but should we have the `result->error_msg = on_run_result.error_msg`?